prime-router/pom.xml

```diff
@@ -583,6 +589,10 @@
                     <groupId>com.github.kittinunf.fuel</groupId>
                     <artifactId>fuel</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>org.jetbrains.kotlinx</groupId>
+                    <artifactId>kotlinx-coroutines-core</artifactId>
+                </exclusion>
             </exclusions>
```

Comment:
I had to update the pom file to tell mock-fuel to not try to fulfill this requirement.